docker/oauthlib/Pipfile

```diff
@@ -9,8 +9,6 @@ verify_ssl = true
 requests-oauthlib = "*"
 jwt = "*"
 pyjwt = "*"
-cryptography = "*"
```

Comment:
why these removed from here? because python3 contains them as core modules?

Reply:
It is included in the base crypto image